PolyEngine/Core/Src/Path.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "Defines.hpp"
+#include "Dynarray.hpp"
+#include "String.hpp"
+
+namespace Poly {
+
+	class CORE_DLLEXPORT Path
+	{
+	public:
+		static Path* Init();
+		//TODO when String.split() will be ready
+		static Path* FromString(const String& source);
+
+		Path& RootAt(const String& root);
```

Comment:
This method is actually unnecessary and system dependant. Dir(..) can be made to handle "C:\" or "/", or we can provide non-static FromString method that let's us do that. i.e.
`Path p; p.FromString("C:\absolute\path\to\resource").Dir("dir1").File("resource.txt");`